CHANGELOG.md

```diff
@@ -18,6 +18,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ### Added
 - Add `EventType::AuthFailed` behind a feature (`auth-failed-event`). Allows the plugin to process
   the [Mullvad VPN specific] event in the fork.
+- Add event types `ClientConnectDefer` and `ClientConnectDeferV2`.
```

Comment:
Can you point out that these are new in OpenVPN 2.5? So it becomes more clear from the changelog that this version of the plugin supports 2.5

Reply:
Done.